advanced_functionality/mxnet_mnist_byom/mxnet_mnist.ipynb

```diff
@@ -290,7 +304,7 @@
    },
    "outputs": [],
    "source": [
-    "sagemaker.Session().delete_endpoint(predictor.endpoint)"
+    "# sagemaker.Session().delete_endpoint(predictor.endpoint)"
```

Comment:
Actually, can you change this back so that it's not commented out?  We've changed our best practice on this to automatically delete temporary example endpoints.